crates/neon-runtime/src/napi/primitive.rs

```diff
@@ -1,10 +1,21 @@
 use raw::{Local, Env};
 
-pub unsafe extern "C" fn undefined(_out: &mut Local) { unimplemented!() }
-
-pub unsafe extern "C" fn null(_out: &mut Local) { unimplemented!() }
-
-pub unsafe extern "C" fn boolean(_out: &mut Local, _b: bool) { unimplemented!() }
+use nodejs_sys as napi;
+
+/// Mutates the `out` argument provided to refer to the global `undefined` object.
+pub unsafe extern "C" fn undefined(out: &mut Local, env: Env) {
```

Comment:
What are your thoughts on making `env` the first argument? I noticed that I was inconsistent in my previous PR.

I think where possible, we should prefer to match the `napi` methods and have it first. What do you think?

Reply:
Yes, I definitely prefer taking `env` as the first parameter, but did it like this to match the API of the few legacy runtime functions that took an `Isolate` as the second parameter. A good followup PR would be to swap everything